ablateLibrary/CMakeLists.txt

```diff
@@ -60,7 +60,7 @@ else()
 endif()
 
 ### Ablate Library Setup ###
-add_library(ablateLibrary SHARED "")
+add_library(ablateLibrary SHARED "" eos/stiffenedGas.cpp eos/stiffenedGas.hpp)
```

Comment:
@maesemen You will want to revert this change and add stiffenedGas.cpp stiffenedGas.hpp to the cmakelist file in the eos folder.